bin/db-initialization.py

```diff
@@ -1,87 +1,101 @@
+    - CONDUCTOR_ROLE_USER and CONDUCTOR_ROLE_PASS:
+          swoop-conductor role username and password
+    - MIGRATION_ROLE_USER and MIGRATION_ROLE_PASS:
+          username and password for migration role
+    - Any additional libpq-supported connection parameters
+          (https://www.postgresql.org/docs/current/libpq-envars.html)
+"""
 import asyncio
 import os
-
-from buildpg import V, asyncpg, render
-
-DB_NAME = os.environ["DATABASE_TO_CREATE"]
-
-
-def db_initialization():
-    async def setup():
-        user = os.environ["POSTGRES_DEFAULT_USERNAME"]
-        password = os.environ["POSTGRES_DEFAULT_PASSWORD"]
-        dsn = f"postgresql://{user}:{password}@postgres:5432/postgres"
-        conn = await asyncpg.connect(dsn=dsn)
-
-        # Create Owner Role
-
-        owner_role_un = os.environ["OWNER_ROLE_USER"]
-
-        q, p = render(
-            """
-                CREATE ROLE :un WITH CREATEDB CREATEROLE LOGIN PASSWORD ':pw';
-            """,
-            un=V(owner_role_un),
-            pw=V(os.environ["OWNER_ROLE_PASS"]),
-        )
-
-        await conn.execute(q, *p)
-
-        # Create Swoop API Role
-
-        q, p = render(
-            """
-                CREATE ROLE :un WITH LOGIN IN ROLE :ir PASSWORD ':pw';
-            """,
-            un=V(os.environ["API_ROLE_USER"]),
-            pw=V(os.environ["API_ROLE_PASS"]),
-            ir=V(owner_role_un),
-        )
-
-        await conn.execute(q, *p)
-
-        # Create Caboose Role
-
-        q, p = render(
-            """
-                CREATE ROLE :un WITH LOGIN IN ROLE :ir PASSWORD ':pw';
-            """,
-            un=V(os.environ["CABOOSE_ROLE_USER"]),
-            pw=V(os.environ["CABOOSE_ROLE_PASS"]),
-            ir=V(owner_role_un),
-        )
-
-        await conn.execute(q, *p)
-
-        # Create Conductor Role
-
-        q, p = render(
-            """
-                CREATE ROLE :un WITH LOGIN IN ROLE :ir PASSWORD ':pw';
-            """,
-            un=V(os.environ["CONDUCTOR_ROLE_USER"]),
-            pw=V(os.environ["CONDUCTOR_ROLE_PASS"]),
-            ir=V(owner_role_un),
-        )
-
-        await conn.execute(q, *p)
-
-        # Create database with owner role as owner
-
-        q, p = render(
-            """
-                CREATE DATABASE :db WITH OWNER :dbo;
-            """,
-            db=V(DB_NAME),
-            dbo=V(owner_role_un),
-        )
-
-        await conn.execute(q, *p)
-
-        await conn.close()
-
-    asyncio.run(setup())
+import sys
+
+import asyncpg
+from buildpg import V, render
+
+from swoop.db import SwoopDB
+
+OWNER_ROLE_NAME = "swoop"
+APPLICATION_ROLES: list[str] = [
+    "API_ROLE",
+    "CABOOSE_ROLE",
+    "CONDUCTOR_ROLE",
+    "MIGRATION_ROLE",
+]
+
+
+def stderr(*args, **kwargs) -> None:
+    kwargs["file"] = sys.stderr
+    print(*args, **kwargs)
+
+
+async def create_owner_role(conn: asyncpg.Connection, owner_role_name: str) -> None:
+    q, p = render(
+        "CREATE ROLE :un",
```

Comment:
By making the owner role not a login role, we don't need any information about this role from the user. This free us up to make the list of application roles passed in from the user, and we could change that list without code changes. Two ways to handle that:

1. use `argparse` and add a "appending" option with `nargs=2` to take in each username/password env var combo
2. support another env var that is the set of username/password env var names, like `SWOOP_DB_ROLES=API_USER:API_PASSWORD,CABOOSE_USER:CABOOSE_PASS`, which could be easily parsed like `[pair.split(":") for pair in os.environ["SWOOP_DB_ROLES"].split(",")]`.